apps/prairielearn/src/batched-migrations/20230913181816_pl_courses_created_at.ts

```diff
@@ -52,10 +52,10 @@ export default makeBatchedMigration({
 
   async execute(start: bigint, end: bigint): Promise<void> {
     for (let id = start; id <= end; id++) {
-      const course = await queryRow(sql.select_course, { course_id: id }, CourseSchema);
+      const course = await queryOptionalRow(sql.select_course, { course_id: id }, CourseSchema);
 
-      if (course.created_at != null) {
-        // The course already has a date; don't overwrite it.
+      if (course == null || course.created_at != null) {
```

Comment:
Shouldn't this be using === and !== for comparisons? 

I thought CI was supposed to check that.

Reply:
We use `eqeqeq`'s `smart` mode, which allows `==`/`!=` when comparing with `null`: https://eslint.org/docs/latest/rules/eqeqeq#smart